database/migrations/2022_02_24_101634_create_users_table.php

```diff
@@ -19,7 +19,7 @@ public function up()
             $table->string('name');
             $table->string('phone');
             $table->string('address');
-            $table->string('email')->unique();
+            $table->string('email',320)->unique();
```

Comment:
tạo migration riêng để update. ko sửa lại file cũ bổ sung cách sau dấu ,
```suggestion
            $table->string('email', 320)->unique();
```